### ISSUE TITLE
JetSmartFilters -> "Search" Filter : allow to search for custom fields AND default fields at the same time

### ISSUE THREAD

**Issue by cookiemachine** [CrocoBlock/suggestions#7665]
desired feature for "Search" filter (SmartFilters):
- allow to search for custom fields AND default fields at the same time

Example:
allow to search for default fields like post title, and also for custom fields like "color"

This is important for better usability for the frontend user.

![search_02](https://github.com/user-attachments/assets/5ccf2ebc-8db9-41ad-9bda-f1935af0a45c)

**Comment by suckedjs** [comment]
Yes, I second this, it will be a very good feature!

**Comment by dantaswebdev** [comment]
Yes. I'm having trouble using query variables in a sentence on the front-end. Example: "Red car 1.0 in Brooklyn". What could be happening?
